lib/xcode/install.rb

```diff
@@ -196,15 +196,13 @@ def get_dmg(version, progress = true, url = nil)
     end
 
     def fetch_seedlist
-      @xcodes = parse_seedlist(spaceship.send(:request, :get, '/services-account/QH65B2/downloadws/listDownloads.action', {
-        start: "0",
-        limit: "1000",
-        sort: "dateModified",
-        dir: "DESC",
-        searchTextField: "",
-        searchCategories: "",
-        search: "false",
-      }).body)
+      @xcodes = parse_seedlist(spaceship.send(:request, :get, '/services-account/QH65B2/downloadws/listDownloads.action', start: '0',
+                                                                                                                          limit: '1000',
+                                                                                                                          sort: 'dateModified',
+                                                                                                                          dir: 'DESC',
+                                                                                                                          searchTextField: '',
+                                                                                                                          searchCategories: '',
+                                                                                                                          search: 'false').body)
```

Comment:
ugh, this is horrible — any way to move the parameters into a visible area and appease rubocop? :)